cdap-examples/LogAnalysis/src/main/scala/co/cask/cdap/examples/loganalysis/ResponseCounterProgram.scala

```diff
@@ -51,7 +51,9 @@ class ResponseCounterProgram extends ScalaSparkProgram {
     val ipCounts = logsData.map(x => (ApacheAccessLog.
       parseFromLogLine(x._2.toString).getIpAddress, 1L)).reduceByKey(_ + _)
 
-    context.writeToDataset(ipCounts, LogAnalysisApp.IP_COUNT_STORE,
-      classOf[String], classOf[java.lang.Long])
+    val outputArgs: util.HashMap[String, String] = new util.HashMap[String, String]
```

Comment:
No need to have type. Can simply be `var outputArgs = new util.HashMap[String, String]`